pages/blog/2023-summary.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Annual Review: State of AsyncAPI Initiative 2023"
+date: 2024-01-18T06:00:00+01:00
+type: Communication
+tags:
+  - Project Status
+cover: /img/posts/2023-summary/blog-banner-2023.webp
+authors:
+  - name: Thulisile Sibanda
+    photo: /img/avatars/thulieblack.webp
+    link: https://www.linkedin.com/in/v-thulisile-sibanda/
+    byline: AsyncAPI Community Manager
+excerpt: '2023 Year in Review'
+featured: true
+---
+
+It's almost unbelievable that we are in 2024. And if you are reading this, Happy New Year to you and your loved ones. Continuing the tradition, we look at how the community has grown over the last 365 days and look forward to our next phase.
+
+If you are curious about our past achievements, [please read the previous summary](https://www.asyncapi.com/blog/2022-summary).
```

Comment:
```suggestion
If you are curious about our past achievements, please [read the previous 2022 AsyncAPI summary](https://www.asyncapi.com/blog/2022-summary).
```

Remember that accessibility in anchor text means clarifying where a link takes you. "Please read the previous summary" doesn't give context and forces the screenreader user to click the link to figure out the content.